SonarQube.Client/Models/ServerSentEvents/ClientContract/ITaintIssue.cs

```diff
@@ -0,0 +1,121 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+
+namespace SonarQube.Client.Models.ServerSentEvents.ClientContract
+{
+    public interface ITaintIssue
+    {
+        //DateTimeOffset CreationDate { get; }
```

Comment:
for some reason I can't deserialize it, but I don't think we need it anyway

Reply:
Need a custom converter maybe?

Reply:
Could be, it works out of the box in our other objects, maybe the format is different on Sq side. Either way, I don't think we need it for now so maybe I'll just not bother with it